### PR TITLE
Bug 1208912 - Fix failure in test_browser_private_navigation.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e0bef458eb104a6e074b7c192dd070594bad93b3

File: tests/python/gaia-ui-tests/gaiatest/apps/search/app.py [hunk 1]
```diff
@@ -39,9 +39,10 @@ def wait_for_history_to_load(self, number_of_items=1):
         Wait(self.marionette).until(lambda m: len(m.find_elements(*self._history_item_locator)) == number_of_items)
 
     def open_new_private_window(self):
-        self.marionette.find_element(*self._private_window_locator).tap()
+        element = self.marionette.find_element(*self._private_window_locator)
+        Wait(self.marionette).until(expected.element_displayed(element))
+        element.tap()
 
         from gaiatest.apps.search.regions.browser import PrivateWindow
         private_window = PrivateWindow(self.marionette)
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/app.py]: Nit: As we don't wait anymore, we can do:
```python
return PrivateWindow(self.marionette)
```

File: tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 class Browser(Base):
 
-    _browser_app_locator = (By.CSS_SELECTOR, 'div.browser[transition-state="opened"]')
+    _browser_app_locator = (By.CSS_SELECTOR, 'div[data-manifest-name="Browser"][transition-state="opened"]')
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py]: This locator is already defined at `gaiatest/apps/search/app.py`. That make me wonder if we shouldn't make `Browser` inherit from `Search`. What do you think @mwargers ?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py]: Yeah, I was thinking the same thing, I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1205262 . I'll needinfo you on this.
We also need to have a way of way of being able to pick the 3rd browser instance when there are 4 browser windows open, for example.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7b7b0748f44d1936a61dce7531c19aee143c9991

**Pull request merged**